src/Symfony/Component/DependencyInjection/Tests/Compiler/AutowirePassTest.php

```diff
@@ -1411,4 +1411,51 @@ public function testAutowireAttributeWithEnvVar()
         $this->assertSame('%env(bool:ENABLED)%', $container->resolveEnvPlaceholders($definition->getArguments()[0]));
         $this->assertSame('%env(default::OPTIONAL)%', $container->resolveEnvPlaceholders($definition->getArguments()[1]));
     }
+
+    /**
+     * @requires PHP < 8.4
```

Comment:
Why would this require PHP < 8.4 ? We want this behavior to work on all supported PHP versions.